### PR TITLE
nvidia-gpu-device-plugin, increase memory limit. 

### PR DESCRIPTION
**Pull request by hangsucho**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug
> /kind design

**What this PR does / why we need it**:
The purpose of this commit is to increase resources.requests.memory and resources.limits.memory.

In docker 18.9.x or later, an error occurs with the following message.
```
Failed create pod sandbox: rpc error: code = Unknown desc = failed to start sandbox container for pod "nvidia-gpu-device-plugin-p9pfj": Error response from daemon: OCI runtime create failed: container_linuxtainer process caused "process_linux.go:297: getting the final child's pid from pipe caused \"read init-p: connection reset by peer
```
It seems to be related to the issue. https://github.com/opencontainers/runc/issues/1914 

If increase resources.requests.memory and resources.limits.memory, works fine.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml [hunk 1]
```diff
@@ -44,10 +44,10 @@ spec:
         resources:
           requests:
             cpu: 50m
-            memory: 10Mi
+            memory: 50Mi
```
Comment (mikedanese) [inline cluster/addons/device-plugins/nvidia-gpu/daemonset.yaml]: This is a substantial increase that will cause eviction of user workload on upgrade.

**Comment by mikedanese** [comment]
> It seems to be related to the issue. [opencontainers/runc#1914](https://github.com/opencontainers/runc/issues/1914)

Can we fix this in runc?

**Comment by spiffxp** [comment]
Ping: is this still needed or being worked?

**Pull request closed without merging**